resources/views/settings/ldap.blade.php

```diff
@@ -63,64 +63,44 @@
                                 {{ Form::label('ldap_integration', trans('admin/settings/general.ldap_integration')) }}
                             </div>
                             <div class="col-md-9">
-                                @if (config('app.lock_passwords')===true)
-                                    {{ Form::checkbox('ldap_enabled', '1', Input::old('ldap_enabled', $setting->ldap_enabled),['class' => 'minimal disabled', 'disabled' => 'disabled']) }}
-                                @else
-                                    {{ Form::checkbox('ldap_enabled', '1', Input::old('ldap_enabled', $setting->ldap_enabled),array('class' => 'minimal')) }}
-                                @endcan
-
+                                {{ Form::checkbox('ldap_enabled', '1', Input::old('ldap_enabled', $setting->ldap_enabled), ['class' => 'minimal '. $setting->demoMode, $setting->demoMode]) }}
```

Comment:
I like this cleanup, a lot, but it definitely ought to be in a separate PR.

I think I might want the helper to me something more like `demoModeElementDisabledAttribute` - or maybe something that's less of a handful to type. But something that explains clearly that it's just the word "disabled" (or maybe `disabled="disabled"`, only for when we are in demoMode.

And I think there are other parts of the app that do those same lookups; it'd be nice to have those in there too.